docker/docker_tools_misc.py

```diff
@@ -69,7 +74,7 @@ def start_servers(dev: bool) -> None:
     Run the web servers -- nginx, web2py, and FastAPI -- used by Runestone. Before starting the server, it will stop any currently-running servers.
     """
 
-    _stop_servers()
+    # _stop_servers()
     _start_servers(dev)
     # If we exit too early, the servers don't start. Perhaps losing the stdout prematurely causes this?
     sleep(7)
```

Comment:
I think this sleep can go now as well since we are starting gunicorn in daemon mode for both servers.